doc/rfcs/0002-support-run-component-without-tiup.md

```diff
@@ -0,0 +1,23 @@
+# Support Run Component without TiUP
+
+## Summary
+
+Allow users to only use TiUP during installation and upgrade, without needing to invoke TiUP while using the component.
+
+## Motivation
+
+- Some users do not like the mandatory update checks and additional outputs by TiUP before invoking the components.
```

Comment:
Could we solve this in a different way without having to get rid of the update checks?  Maybe we can do the upgrade checks in a separate goroutine and then store the information about available versions on disk and check this data on the next startup?

I think the geographical location might also matter in how fast or slow the check is depending on which mirror is chosen.

Reply:
It is a possible way, but some users just hate the additional output message